src/data/faq_de.json

```diff
@@ -329,13 +329,13 @@
                         "active": "true",
                         "textblock": [
                             "<ul>",
-                            "<li><a href='#vac_cert_invalid_211111' target='_self'>Warum erhalte ich die Meldung, dass mein Zertifikat neu ausgestellt werden muss?</a></li><li><a href='#vac_cert_booster_info' target='_self'>Wie erfahre ich von Auffrischungsimpfungen im Kontext von Corona?</a></li><li><a href='#vac_cert_booster_cwa' target='_self'>Wie verwalte ich Auffrischungsimpfungen in der CWA?</a></li><li><a href='#vac_booster_notification' target='_self'>Warum erhalte ich eine Benachrichtigung über eine Impfempfehlung der Ständigen Impfkomission (STIKO)?</a></li>",
+                            "<li><a href='#vac_cert_invalid_21_1' target='_self'>Warum erhalte ich die Meldung, dass mein Zertifikat neu ausgestellt werden muss?</a></li><li><a href='#vac_cert_booster_info' target='_self'>Wie erfahre ich von Auffrischungsimpfungen im Kontext von Corona?</a></li><li><a href='#vac_cert_booster_cwa' target='_self'>Wie verwalte ich Auffrischungsimpfungen in der CWA?</a></li><li><a href='#vac_booster_notification' target='_self'>Warum erhalte ich eine Benachrichtigung über eine Impfempfehlung der Ständigen Impfkomission (STIKO)?</a></li>",

```

Comment:
@dsarkar Does this need to be changed in English too? Should I quickly provide a PR or will you do so?

Reply:
@Ein-Tim 
It needs changes because otherwise it is broken.

Reply:
@MikeMcC399 

For me it still works (maybe because I've set my browser to always open links in new tabs?). But I'll provide a PR fixing this in a few minutes.